src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java

```diff
@@ -1,16 +1,10 @@
 package gov.usgs.wma.waterdata.groundwater;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
-
+import com.fasterxml.jackson.core.*;
 import org.springframework.util.StringUtils;
 
-import com.fasterxml.jackson.core.JsonFactory;
-import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonToken;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Sometimes intellij will contract your imports to use the *, I think for most projects we opt not to do this so it's easier to see exactly which imports we're using.